packages/hemera-nsq/package.json

```diff
@@ -15,9 +15,8 @@
   "author": "Dustin Deus (https://github.com/StarpTech)",
   "license": "MIT",
   "dependencies": {
-    "hemera-joi": "^2.0.1",
     "hemera-plugin": "^0.0.14",
-    "nsqjs": "0.7.x"
+    "nsqjs": "0.9.x"
```

Comment:
nsqjs 0.9 dropped support for node < 6 please restrict the node engine and add it to the docs.